tidb/run

```diff
@@ -0,0 +1,6 @@
+lein run test \
```

Comment:
one script is enough, it's modifiable, we can choose which test to run and which nemesis to use, etc.